### PR TITLE
[TT-415] keepers docker migration

### PR DESCRIPTION
**Pull request by tateexon**

### CODE REVIEW

File: integration-tests/smoke/keeper_test.go [hunk 1]
```diff
@@ -120,11 +99,11 @@ func TestKeeperBasicSmoke(t *testing.T) {
 			)
 			gom := gomega.NewGomegaWithT(t)
 
-			actions.CreateKeeperJobs(t, chainlinkNodes, registry, contracts.OCRv2Config{})
+			actions.CreateKeeperJobsLocal(t, chainlinkNodes, registry, contracts.OCRv2Config{})
```
Comment (skudasov) [inline integration-tests/smoke/keeper_test.go:102]: @tateexon Please, also, remove `*testing.T` from the helpers. You can use `errors.Wrap(err, ...)` to make it more descriptive.

File: integration-tests/smoke/keeper_test.go [hunk 2]
```diff
@@ -1103,71 +1058,43 @@ func TestKeeperUpdateCheckData(t *testing.T) {
...
-		&environment.Config{
-			NamespacePrefix: fmt.Sprintf("smoke-keeper-%s-%s", testName, networkName),
-			Test:            t,
-		}).
-		AddHelm(mockservercfg.New(nil)).
-		AddHelm(mockserver.New(nil)).
-		AddHelm(evmConfig).
-		AddHelmCharts(chainlinkChart)
-	err = testEnvironment.Run()
+	// TODO: Add ability to use non simulated network
```
Comment (skudasov) [inline integration-tests/smoke/keeper_test.go:1072]: @ilija42 is currently working on enabling both load and smoke tests to run on multiple networks. I think you can skip implementing it in your branch @tateexon https://github.com/smartcontractkit/chainlink/pull/10178

Comment (ilija42) [reply integration-tests/smoke/keeper_test.go:1072]: You were probably thinking about @iljapavlovs  :D

Comment (skudasov) [reply integration-tests/smoke/keeper_test.go:1072]: Yep, thanks! :D

File: integration-tests/smoke/keeper_test.go (hunk 1 above)
Comment (tateexon) [reply integration-tests/smoke/keeper_test.go]: Updated in commit https://github.com/smartcontractkit/chainlink/pull/10155/commits/dc51b12bbab6bee6f8637cab61cc0f7d6b8e4dc6

**Review by skudasov — APPROVED** [review]

**Pull request merged**